tsconfig-webpack.json

```diff
@@ -0,0 +1,8 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
```

Comment:
Should this be ESNext? I have no idea about this stuff

Reply:
no. This file is explicitly for `webpack.config.ts`   - it will not work at all if its not `commonjs`

https://webpack.js.org/configuration/configuration-languages/#typescript